doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,164 +3,239 @@
+Tasks that have `ioSpecification` should always define an `inputSet` and `outputSet`, even if those are empty.
 
 Steps to reproduce:
 
-. Create a KJAR that contains a PMML asset.
-. Start `kie-server-spring-boot-starter` on Windows.
-. Deploy the KJAR to a {KIE_SERVER} that is running the `kie-server-spring-boot-starter` Spring Boot starter on Microsoft Windows.
+Create a user task with no output mapping. This will generate the following output:
 
-Expected result: The KJAR is deployed successfully.
+[source]
+----
+<bpmn2:ioSpecification id="_rEnOQY5AEem2-an8Fwu06w">
+<bpmn2:dataInput id="38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_TaskNameInputX" drools:dtype="Object" itemSubjectRef="_38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_TaskNameInputXItem" name="TaskName"/>
+<bpmn2:dataInput id="38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_SkippableInputX" drools:dtype="Object" itemSubjectRef="_38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_SkippableInputXItem" name="Skippable"/>
+<bpmn2:inputSet id="_rEnOQo5AEem2-an8Fwu06w">
+<bpmn2:dataInputRefs>_38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_TaskNameInputX</bpmn2:dataInputRefs>
+<bpmn2:dataInputRefs>_38B29C0C-6BD8-4275-ACCE-D5D97DEBB2CB_SkippableInputX</bpmn2:dataInputRefs>
+</bpmn2:inputSet>
+</bpmn2:ioSpecification>
+----
 
-Actual result: The KJAR cannot be deployed.
+Expected result: The output includes an empty `outputSet` attribute.
+
+Actual result: The output does not include an `outputSet` attribute.
 
 Workaround: None.
 
-ifdef::DM[]
-== OpenShift
 
-.User password is displayed in the Readiness and Liveness probes [https://issues.jboss.org/browse/RHDM-848[RHDM-848]]
+.Service Task is not usable at runtime [https://issues.jboss.org/browse/JBPM-8581[JBPM-8581]]
+
+Issue: If you execute a Java method with the current implementation of the General Service Task, you will encounter the following limitations:
+
+* It is not possible to send data to the java method you are invoking.
+* It is not possible to receive data from the java method you invoked.
+* It is possible to chose a Web service to call in a general service task, but it is not possible to use it at runtime.
+* If you have properly configured a service task in the legacy process designer, many properties will be lost and it will not be possible to use Service Task at runtime after migration:
 
-Issue: If you deploy {PRODUCT} on OpenShift, the user password is displayed in both the Readiness and Liveness probes.
 
 Workaround: None.
-endif::DM[]
 
-ifdef::PAM[]
-== OpenShift
+.Nodes and distribution lines do not fully align horizontally [https://issues.jboss.org/browse/JBPM-8555[JBPM-8555]]
+
+Issue: If you drag a node, the align and distribution mechanism correctly displays the alignment lines and also applies the snap-to-line behavior when the mouse moves. However, it is not possible to align the nodes horizontally which means that it is difficult to have the connectors precisely aligned horizontally.
```

Comment:
Hey @emmurphy1 
Probably I would move the first sentence ("_If you drag [...] when the mouse moves_") out from the `issue` section? I mean, that's what the ticket also describes but not sure if it's clear, that's just explaining this functionality, there is no "issue" around this. Anyway just giving my opinion.. up to you how to make it better :)

On the other hand, it is correct than is not possible to align the nodes, but I would probably also add the word "some", meaning that not ALL nodes are not being aligned properly - eg: it is possible to align tasks when same size, but it's hard when mixing tasks with events or gateways...  Also the lines that appear, to help on the alignment while dragging, are not well aligned horizontally, I think it's the same bug causing both wrong behaviors....

Thanks!

Reply:
Updated, please review.

Reply:
it looks good to me, thx :+1: